src/main/resources/tomcat-eureka-config/dbmigration/registry-changelog-ec-i2b2-integration-config.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<!--
+  #%L
+  Eureka! Clinical Registry Service
+  %%
+  Copyright (C) 2017 - 2018 Emory University
+  %%
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  
+       http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  #L%
+  -->
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                         http://www.liquibase.org/xml/ns/dbchangelog
+                         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd"
+		   logicalFilePath="changelog-master.xml">
+    <changeSet author="arpost" id="EC-I2B2-INTEGRATION-1">
```

Comment:
These need to be just the components that are started when you run embedded tomcat, and the URLs need to be the actual URLs where they are found.